source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -12,26 +12,15 @@ section has more detail on how we currently use Node.js at GDS and why this
 decision has been made.
 
 ## Guidance
-The guidance set out here should be followed in conjunction with the advice 
-on the main [programming languages manual page](../../programming-languages.html).
+The guidance set out here should be followed in conjunction with the advice on the main [programming languages manual page](../../programming-languages.html).
 
 If you want to add guidance on Node.js please see [Updating this manual](#updating-this-manual) below.
 
 ### Style
 #### Formatting and linting
-To maintain consistent and readable style we use a few tools to keep our JavaScript 
-consistent. We use [Standard] style guide and linter.
-To make sure everything stays clean we recommend running these tools pre-commit. 
-It is possible to setup a pre-commit hook to automate this process.
-[Standard] can auto-fix common errors and have clear error messaging when auto-fix 
-isn’t possible. 
-
-We’ve encapsulated our recommended process into a [repo](https://github.com/jonheslop/nodejs-starter) where everything 
-is controlled by a few lines of config within your `package.json`. Once you have 
-run `npm install`, any time you make a commit [Standard] runs which formats and 
-lints your code. [Standard] will autofix small errors then continue with your 
-commit. If there’s something it can’t fix then the commit fails with a useful 
-error message such as:
+To maintain a consistent and readable JavaScript codebase we use [StandardJS]. This includes a built in formatter, linter and automatic code fixer (auto-fix). To make sure everything stays clean we recommend running this tool before each commit. It is possible to setup a pre-commit hook to automate this process. When [StandardJS] auto-fix's errors it has clear and concise messaging and it also tells you when an automatic fix isn't possible.
+
+We’ve encapsulated our recommended process into a separate [repository](https://github.com/jonheslop/nodejs-starter) where everything is controlled by a few lines of config within your `package.json` file. Once you have run `npm install`, any time you make a commit, [StandardJS] runs. It will then format and lint your code automatically. [StandardJS] will auto-fix obvious errors then continue with your commit. If there’s something it can’t fix, then the commit fails with a useful error message such as:
```

Comment:
Should we not move jonheslop/nodejs-starter to alphagov

Reply:
Yes definitely, but let’s not block this PR, I'll raise a separate issue for that. I can ask David/Dom/Will if they can create a repo for me.

Reply:
Cool, lets get that done. Approved on that assumption